src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    public static final double CONSTANT = 0.5;
+    private double bisector;
+    private double side;
+
+    public RightTriangle(Color color, double side, double bisector) {
+        super(color);
+        this.side = side;
+        this.bisector = bisector;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return side * bisector * CONSTANT;
```

Comment:
```suggestion
        return side * bisector * 0.5;
```
the constant is not informative